tox.ini

```diff
@@ -38,6 +38,8 @@ deps =
   sphinx
   -r{toxinidir}/docs/requirements-docs.txt
 commands = make -C docs html latex
+whitelist_externals =
+  make
```

Comment:
I can? It's [standard tox behavior][1], though.

  [1]: http://tox.readthedocs.io/en/latest/example/basic.html